.github/workflows/build-linux.yml

```diff
@@ -89,5 +89,8 @@ jobs:
         run: |
           sudo apt-get -y install libboost-all-dev
 
-      - name: Build and Test
+      - name: Build
```

Comment:
There isn't quite a need to separate this since running the tests will automatically trigger the build.

Reply:
ah, whoops, i had split them to make it easier to read.  i put it back.